documentation/book/proc-upgrading-brokers-newer-kafka.adoc

```diff
@@ -18,7 +18,7 @@ This procedure describes how to upgrade a {ProductName} Kafka cluster from one v
 
 . Consult the table below and determine whether the new Kafka version has a different log message format version than the previous version. 
 +
-include::snip-kafka-versions.adoc[leveloffset=+1]
+{blank}include::snip-kafka-versions.adoc[leveloffset=+1]
```

Comment:
What does `{blank}` do?

Reply:
I'm not 100% sure. But it seems to help AsciiDoctor to render the numbers correctly. Without it, I get the same problem with the rendered `+` characters.

For background, see this [forum post](http://discuss.asciidoctor.org/source-block-on-same-line-as-list-item-td5065.html#a5125).